x/bank/exported/exported.go

```diff
@@ -1,14 +1,15 @@
 package exported
 
 import (
+	"cosmossdk.io/core/address"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 )
 
 // GenesisBalance defines a genesis balance interface that allows for account
 // address and balance retrieval.
 type GenesisBalance interface {
-	GetAddress() sdk.AccAddress
+	GetAddress(address.Codec) ([]byte, error)
```

Comment:
this is causing a lot of breakage, not sure how to avoid it. Anyone have ideas

Reply:
Why not handle this logic in the genesis functions and return string here?